app/models/solr_document.rb

```diff
@@ -57,6 +57,10 @@ def etd?
     self['has_model_ssim'] == [ETD.to_class_uri]
   end
 
+  def admin_policy_id
+    fetch('isGovernedBy_ssim').first
```

Comment:
No, I want this to fail noisily if you call it on something without an admin policy.